sonar-db/src/main/java/org/sonar/db/user/RoleMapper.java

```diff
@@ -42,9 +42,9 @@
 
   void insertGroupRole(GroupRoleDto groupRole);
 
-  void insertUserRole(UserRoleDto userRole);
+  void insertUserRole(UserPermissionDto userRole);
 
-  void deleteUserRole(UserRoleDto userRole);
+  void deleteUserRole(UserPermissionDto userRole);
 
   void deleteGroupRole(GroupRoleDto groupRole);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)